lec15-data-viz.Rmd

```diff
@@ -1,4 +1,241 @@
+
 ---
-title: "Data visualization in R"
+title: Data visualization in R
+author: Lindsay Coome
 ---
 
+## Lesson preamble
+
+> ### Learning Objectives
+>
+> - Making better plots for outreach and communication.
+> - Review how to produce scatter plots with trendlines using ggplot.
+> - Make bar and other graphs using ggplot.
+> - Advanced faceting and segmenting data.
+> - Altering asthetic features in ggplot.
+> - Making interactive and quick graphs.
+> 
+> ### Lesson outline
+>
+> - How to visualize data (20 min)
+> - ggplot basics (10 min)
+> - Basics and faceting (15 min)
+> - Aesthetics and plot types (20 min)
+> - Interactive and quick graphing (10 min)
```

Comment:
Not included in this lesson, did you want to include this in the next lesson? or still plan on adding it?

Reply:
which part??

Reply:
the interactive part? Unless I missed something though, sorry if that's the case.